### DISCUSSION TITLE
设计建议

### DISCUSSION THREAD

**Discussion by dukeluo** [dukeluo/wai#91] [Ideas]
我理想中的「歪脖子」一定是好看的，好看才能留住注意力，多扭动几下脖子。未来「歪脖子」的路径，一定是不断向好看的目标优化的。奈何我本是开发，设计素养有限，只知道让「歪脖子」更加的方正。如果你有好的设计想法，欢迎在此留言，比如：

- 好看的字体
- 优秀的布局
- 引人注目的文案
- 眼前一亮的配色
- 契合的背景动画
...

谢谢大家！

**Comment by Camusugar** [comment]
很喜欢这个插件，简约又有意思，还能涨知识。但是每次开新页面的时候，模式设置太显眼了，而且还是动画的，希望模式设置可以变成一个小小的齿轮，这样开新页面之后不会被它吸引注意力：） 感谢开发者，笔芯

**Reply by dukeluo** [reply]
谢谢你的喜欢，后续计划有专门的设置菜单，这个配置会挪进去，设置菜单入口不出意外会是一个齿轮

**Comment by Luyuxiang-Hi** [comment]
插件太好用了，希望可以加入搜索功能，习惯在新标签页的大搜索框搜东西，没了好不习惯

**Reply by dukeluo** [reply]
搜索框的话，它肯定不能转，然后它怎么和现有的布局结合起来，显得和谐，这是要考虑的，目前还没好的想法。另外，我想浏览器地址栏也可以直接搜东西吧。

**Comment by luckyporo** [comment]
非常喜欢您的插件，希望可以开放模块自定义功能，感觉历史上的今天、这个季节吃什么两个模块有点单薄容易觉得无聊，希望可以加入TODO LIST和常用页面这两个模块，这样感觉更实用一些，感谢~

**Reply by dukeluo** [reply]
常用页面这个未来会有。TODO LIST 的话有想过，但我不想实现一个 TODO 功能，我有一个想法是集成其它 TODO 服务，获取它数据，只做展示。啥 TODO 用得比较多呢 🧐

**Reply by luckyporo** [reply]
根据身边统计学来说用的最多是滴答清单
我目前在用microsoft todo

**Reply by idelin** [reply]
可以考虑下滴答清单
https://developer.dida365.com/

**Reply by whitewatercn** [reply]
> 根据身边统计学来说用的最多是滴答清单 我目前在用microsoft todo

滴答清单有非常完备的网页端，可以直接嵌进去用

**Comment by gagajtajw** [comment]
或许考虑加入夜间模式？

**Reply by whitewatercn** [reply]
同求

**Comment by joey5403** [comment]
引人注目的文案，可以使用头条热榜和微博热搜，八卦才是源动力。
